Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -736,6 +742,13 @@ extension Purchases {
 
 public extension Purchases {
 
+    /// Parses a deep link URL to verify it's a RevenueCat supported DeepLink
+    /// and returns the RevenueCat deep link for that URL.
+    /// - Seealso: ``Purchases/redeemWebPurchase(_:)``
```

Comment:
```suggestion
    /// - See also: ``Purchases/redeemWebPurchase(_:)``
```

Reply:
Actually I think `seealso` is correct? At least the IDE seems to process that correctly.

Reply:
oh TIL it's a keyword for the IDE!